packages/discord.js/typings/index.d.ts

```diff
@@ -1642,6 +1642,38 @@ export class GuildMember extends Base {
   public valueOf(): string;
 }
 
+export class UncachedGuildMember extends Base {
```

Comment:
It doesn't extend just base, you probably need to extend similar to GuildMember/User

Reply:
the GuildMember class definition also extends base, but that also has an interface so ig that's what's missing? I'm not sure why that has both an interface and a class

Reply:
We probably use the interface to make mixins easier, not sure 😓

Reply:
Yep, it‘s the improved mixin typings using interface merging. The old Mixins were a bad fit for documentation purposes